x-pack/plugins/encrypted_saved_objects/server/saved_objects/encrypted_saved_objects_client_wrapper.ts

```diff
@@ -190,6 +191,28 @@ export class EncryptedSavedObjectsClientWrapper implements SavedObjectsClientCon
     );
   }
 
+  public async bulkResolve<T = unknown>(
+    objects: SavedObjectsBulkResolveObject[],
+    options?: SavedObjectsBaseOptions
+  ) {
+    const bulkResolveResult = await this.options.baseClient.bulkResolve<T>(objects, options);
+
+    for (const resolved of bulkResolveResult.resolved_objects) {
+      const savedObject = resolved.saved_object;
+      await this.handleEncryptedAttributesInResponse(
```

Comment:
I did this to stay consistent with the `handleEncryptedAttributesInBulkResponse` function.

We could use `Promise.all` but I'm not sure there's any benefit, since these aren't network requests or anything, we're really just waiting for each object to be decrypted and Node can only do one at a time because it's single-threaded.